charts/kyuubi/values.yaml

```diff
@@ -116,13 +116,14 @@ server:
       nodePort: ~
       annotations: {}
 
+monitoring:
   # Exposes metrics in Prometheus format
   prometheus:
     enabled: true
     port: 10019
     service:
```

Comment:
Could you please check if the node `service` is not used anymore and deleted it?

Reply:
@dnskr  I think "monitoring.prometheus.service" cannot be deleted because this node is used to control whether the prometheus service of kyuubi-headless is displayed.

Reply:
Only `monitoring.prometheus.enabled` and `monitoring.prometheus.port`  are used as I can see. [charts/kyuubi/templates/kyuubi-headless-service.yaml](https://github.com/apache/kyuubi/pull/5281/files#diff-a15690f05b8022769e74321d33a3f77de06a647df4f0d5c34aad721a9a92bb6a)
```yaml
    {{- if .Values.monitoring.prometheus.enabled }}
    - name: prometheus
      port: {{ .Values.monitoring.prometheus.port }}
      targetPort: {{ .Values.monitoring.prometheus.port }}
    {{- end }}
```

Reply:
@dnskr  After removing, how to control the port of Prometheus and whether to apply it? If the default parameters that cannot be changed are set, I think it is not a good solution.

Reply:
@dnskr  Are you still following this PR?

Reply:
Sorry for the delay, I missed your last comment.
I think that Prometheus metrics endpoint doesn't require dedicated `Service` and I don't remember I saw it in other Helm charts. So it's fine that you deleted it and we have `prometheus` port in Kyuubi headless service only.

In the current implementation `{{ .Values.monitoring.prometheus.port }}` value is used for `prometheus` port of the container and `Service` ports (port and targetPort), which is also fine. So, if none of `{{ .Values.monitoring.prometheus.service.* }}` values are used, I would suggest to delete them for now and add them later if needed.

Reply:
OK,